drivers/include/mtd.h

```diff
@@ -176,6 +232,24 @@ int mtd_init(mtd_dev_t *mtd);
  */
 int mtd_read(mtd_dev_t *mtd, void *dest, uint32_t addr, uint32_t count);
 
+/**
+ * @brief   Read data from a MTD device with pagewise addressing
+ *
+ * @param      mtd   the device to read from
+ * @param[out] dest  the buffer to fill in
+ * @param[in]  page  the start page to read from
+ * @param[in]  offset offset from the start of the page (in bytes)
+ * @param[in]  count the number of bytes to read
```

Comment:
Missing indent/alignment of of the descriptions after the parameter name. More instances below.